bin/pint

```diff
@@ -20,6 +20,15 @@
 #
 """
 usage: pint -h | --help
+       pint providers
+          [ --json | --xml ]
+       pint image_states
+          [ --json | --xml ]
+       pint (amazon|google|microsoft|oracle) server_types
```

Comment:
Given that the server now tells us the available providers there is no longer a need to keep a hard coded list of the provider in the client code. So the list needs to be generated.